src/pocketmine/block/Stone.php

```diff
@@ -77,12 +77,18 @@ public function getBreakTime(Item $item){
 
 	public function getDrops(Item $item){
 		if($item->isPickaxe() >= Tool::TIER_WOODEN){
-			return [
-				[Item::COBBLESTONE, 0, 1],
```

Comment:
Why not simply replace this line into `return [[Item::COBBLESTONE, $this->meta, 1]];`?